projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -69,6 +70,13 @@ public static function load_coming_soon() {
 		}
 	}
 
+	/**
+	 * Load the Launchpad feature.
+	 */
+	public static function load_launchpad() {
+		require_once __DIR__ . '/features/launchpad/launchpad.php';
```

Comment:
We can keep this `/features/launchpad/launchpad.php` import and eventually remove the same import from the constructor in `class-wpcom-rest-api-v2-endpoint-launchpad-checklist.php`